spec/asset_spec.rb

```diff
@@ -120,4 +120,26 @@
       end
     end
   end
+
+  describe 'issues' do
```

Comment:
Instead of `issues`, would say `serialization` be preferable? Just thinking that having loads and loads of issues added here might not be so maintainable.